plonetheme/barceloneta/theme/less/contents.plone.less

```diff
@@ -37,4 +37,12 @@ body#visual-portal-wrapper.pat-plone #edit-zone {
 	.contenttype-collection:after {	content: '\e808';}
 	.contenttype-event:after {		content: '\e809';}
 	.contenttype-news-item:after {	content: '\e80f';}
-}
\ No newline at end of file
+}
+
+//*// thumbnails for images, leadimages  in listings and portlets
+    .image-icon, .image-tile {
+      float:right;
+}
+// to supress thumbnails in listings and/or portlets uncomment the following
```

Comment:
In docs additionally to this comment here... I like CSS comments like this here. Normally CSS files are heavily underdocumented.